bitfield/models.py

```diff
@@ -28,9 +28,19 @@ def __iter__(self):
             yield flag
 
     def __getattr__(self, key):
-        if key not in self._flags:
+        if key == '_flags':
+            # Since __getattr__ is for fallback, reaching here from Python
+            # means that there's no '_flags' attribute in this object,
+            # which may be caused by intermediate state while copying etc.
+            raise AttributeError(
```

Comment:
What's your thinking on whether AttributeError is the right exception to raise here?

Reply:
@timabbott Commit has been updated to include the explanation and rebased on master.